### PR TITLE
Remote Inspector: fix remote inspector.

### PR DESCRIPTION
**Pull request by wouterlucas**

### CODE REVIEW

File: src/tests/webkit/WebKit_ChannelChange_001.test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  steps: [
+    {
+      description: 'Attach to the logs to capture the log output and run the test',
+      repeat: 30,
+      sleep: 5,
+      test() {
+        return new Promise((resolve, reject) => {
+          let hostIP = this.$thunder.api.options.host
+
+          function parseGoogleLogs(error, log) {
```
Comment (wouterlucas) [inline src/tests/webkit/WebKit_ChannelChange_001.test.js]: we might want to rename this to something more generic :)

File: src/tests/webkit/WebKit_ChannelChange_001.test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          logger = new AttachToLogs(parseGoogleLogs, hostIP)
+          logger.connect()
+          setWebKitUrl.call(this, url)
+        })
+      },
+    },
+  ],
+  validate(res) {
+    //TODO Need to update this line
+    if (res > 1000) {
```
Comment (wouterlucas) [inline src/tests/webkit/WebKit_ChannelChange_001.test.js]: I realize that the current html only goes by count, we may need to modify that to add support for time based?

File: src/tests/youtube/Youtube_EME_001.test.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ import { AttachToLogs } from '../../commonMethods/remoteWebInspector'
 let url =
   'https://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2018.html?test_type=encryptedmedia-test&enablewebm=off&command=run'
```
Comment (wouterlucas) [inline src/tests/youtube/Youtube_EME_001.test.js]: Should we keep this test in? I dont think that url works anymore right?

File: src/tests/webkit/WebKit_ChannelChange_001.test.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/tests/webkit/WebKit_ChannelChange_001.test.js]: Does this mean the HTML you created needs modifications related to time(currently 5 minutes) such that it runs around 1000 times or for specific amount of time.

File: src/tests/youtube/Youtube_EME_001.test.js (hunk 3 above)
Comment (sandeep-vedam) [reply src/tests/youtube/Youtube_EME_001.test.js]: Yes the URL doesn't work and this test is no more in master.

File: src/tests/webkit/WebKit_ChannelChange_001.test.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/tests/webkit/WebKit_ChannelChange_001.test.js]: Agreed. Will rename this function.

**Pull request merged**